hammr/commands/template/template.py

```diff
@@ -587,6 +587,7 @@ def import_stack(self, file, isImport, isForce, rbundles, isUseMajor):
                             printer.out(status.detailedErrorMsg)
                     else:
                         printer.out("Template create: "+status.message+"\n"+status.errorMessage, printer.ERROR)
+                        return 2
```

Comment:
could be for the exit code of the block before also, isn't it?

Reply:
Yes